product_docs/docs/pgd/4/rel_notes/pgd_4.3.0_rel_notes.mdx

```diff
@@ -0,0 +1,15 @@
+---
+title: "Release notes for EDB Postgres Distributed version 4.3.0"
+navTitle: "Version 4.3.0"
+---
+
+EDB Postgres Distributed version 4.2.2 is a patch release of EDB Postgres Distributed 4, which includes bug fixes for issues identified in previous versions.
+
+| Component | Version |  Type   |                                                   Description                                                           |
+| --------- | ------- | ------- | ------------------------------------------------------------------------------------------------------------------------|
+| BDR       | 4.3.0   | Bug fix | Handle ALTER TABLE...ALTER COLUMN...TYPE...on a non-data node (BDR-2768 RT86808) <p> A global DML lock is required if a table rewrite is needed, however this cannot be taken if the query is executed on a non-data node (logical standby or subscriber-only). </p> |
+| BDR       | 4.3.0   | Feature | Track autopartition leader node (BDR-2721) <p> Add a catalog table that can be used for tracking the leader node </p> |
+| BDR       | 4.3.0   | Feature | Add node_kind to bdr.node <p> We can check the bdr.node what is the kind of a node </p> |
+| BDR       | 4.3.0   | Feature | Implement bdr.alter node_kind() <p> Function to change a node kind </p> |
```

Comment:
This does not need separate These two just enable the autopatition leader thing so I would not mentione them separately